custom-fonts.md

```diff
@@ -59,6 +58,42 @@ var buttonText = const Text(
   style: textStyle,
 );
 ```
+### Fonts in package dependencies {#from-packages}
+
+To use a font family defined in a package, the `package` argument must be provided. For instance, suppose the font
+declaration above is in the `pubspec.yaml` of a package named `my_package` which the app depends on. Then creating the
+TextStyle is done as follows:
+
+```dart
+const textStyle = const TextStyle(
+  fontFamily: 'Raleway',
+  package: 'my_package',
+);
+```
+
+This is also how the package itself should create the style.
+
+A package can also provide font files in its `lib/` folder which will not automatically be 
```

Comment:
When you say also here, what is the other alternative the also is referring to? Is it putting fonts outside the 'lib' folder and some different behaviour arises? Or not needing to refer to the package's fonts specifically in the app's yaml (like the text above seems to suggest since you didn't say pubspec changes in the app were needed before consuming via const TextStyle(...

Reply:
It's not needing to refer to it in the app's yaml. 

If the package declares a font family in its yaml the app can consume as shown in the example since the relevant font files will be bundled with the app. Another use case is a package that has several font files but doesn't necessarily declare a font family in its yaml. Then it's up to the app to declare a font family and choose the relevant files.